TAO/tao/Strategies/DIOP_Acceptor.cpp

```diff
@@ -393,27 +394,59 @@ int
 TAO_DIOP_Acceptor::open_i (const ACE_INET_Addr& addr,
                            ACE_Reactor *reactor)
 {
-  ACE_NEW_RETURN (this->connection_handler_,
-                  TAO_DIOP_Connection_Handler (this->orb_core_),
-                  -1);
+  unsigned short const requested_port = addr.get_port_number ();
+  ACE_UINT32 last_port = requested_port + this->port_span_ - 1;
+  if (last_port > ACE_MAX_DEFAULT_PORT)
```

Comment:
Could use ACE_MIN here which could make last_port const

Reply:
Great suggestion. Done.

Reply:
Can you also update SCIOP_Acceptor.cpp and SSLIOP_Acceptor with the similar change?

Reply:
Done.